src/sentry/migrations/0396_add_usecase_to_indexer.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 2.2.28 on 2023-03-16 22:02
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0395_add_index_to_date_added"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                    ALTER TABLE "sentry_perfstringindexer" ADD COLUMN "use_case_id" varchar(120) NOT NULL DEFAULT 'performance';
```

Comment:
I understand why we need the default for now, but we can drop it later, right?

Reply:
You can, but is there any reason to?

Reply:
There's a potential future where performance is not the most relevant use case on this database. It's not extremely important but [once we have the application logic on top] I'd slightly prefer we fail to insert a row vs. insert a row with the performance use-case without knowing it.